src/upgrade/check.rs

```diff
@@ -3,24 +3,46 @@ use ferinth::structures::version::{Version, VersionFile};
 use furse::structures::file_structs::File;
 use octocrab::models::repos::{Asset, Release};
 
+fn is_jar_file(asset_name: &str) -> bool {
+    asset_name.contains("jar")
```

Comment:
`main` currently changed this to check that the filename ends with `.jar` instead. Just something to keep in mind when merging.